spring-cloud-gcp-pubsub-stream-binder/src/test/java/org/springframework/cloud/gcp/stream/binder/pubsub/PubSubMessageChannelBinderTests.java

```diff
@@ -25,34 +26,30 @@
 import org.springframework.cloud.stream.binder.ExtendedProducerProperties;
 import org.springframework.cloud.stream.binder.Spy;
 
-import static org.assertj.core.api.Assumptions.assumeThat;
+import static org.junit.Assume.assumeTrue;
 
 /**
- * Integration tests that require the Pub/Sub emulator to be installed and only run if the
- * GCP_PUBSUB_EMULATOR envvar is present.
+ * Integration tests that require the Pub/Sub emulator to be installed.
  *
  * @author João André Martins
+ * @author Elena Felder
  */
-public class PubSubMessageChannelBinderTests extends AbstractBinderTests<PubSubTestBinder,
-		ExtendedConsumerProperties<PubSubConsumerProperties>,
-		ExtendedProducerProperties<PubSubProducerProperties>> {
+public class PubSubMessageChannelBinderTests extends
+		AbstractBinderTests<PubSubTestBinder, ExtendedConsumerProperties<PubSubConsumerProperties>,
+				ExtendedProducerProperties<PubSubProducerProperties>> {
 
-	private static final String EMULATOR_HOST_ENVVAR_NAME = "PUBSUB_EMULATOR_HOST";
-
-	private PubSubTestBinder binder;
-
-	public PubSubMessageChannelBinderTests() {
-		this.binder = new PubSubTestBinder(System.getenv(EMULATOR_HOST_ENVVAR_NAME));
-	}
+	@ClassRule
+	public static PubSubEmulator emulator = new PubSubEmulator();
 
 	@BeforeClass
 	public static void enableTests() {
-		assumeThat(System.getenv(EMULATOR_HOST_ENVVAR_NAME)).isNotNull();
+		assumeTrue(
```

Comment:
This still doesn't work for me since the `@ClassRule` is applied first.

I think it doesn't matter if we are in `it` mode or not, since there is just no `gcloud` software.

Can we be protected against that?

Thanks